CHANGES/298.feature

```diff
@@ -0,0 +1 @@
+Add feature flag to enable/disable execution environments
```

Comment:
Here and in the commit message, I would make sure to explicitly include GALAXY_FEATURE_FLAGS in the messages.
The reason being that at some point, someone will be working on something related and will end needing to know when GALAXY_FEATURE_FLAGS setting was introduced. And adding a more greppable string in the notes will make it easier to find.